common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -100,6 +109,26 @@
                 }
             };
 
+            NewPostView.prototype.getGroupName = function() {
+                var userGroupId;
+                var cohort;
+                var group_name = null;
+                if (this.course_settings.get('is_cohorted')) {
+                    userGroupId = $('#discussion-container').data('user-group-id');
+                    if (!userGroupId) {
+                        userGroupId = this.user_group_id;
```

Comment:
I'm curious why we sometimes get this from the data attribute and other times from `this` ?